pkg/controller/tiers/tiers_controller_test.go

```diff
@@ -20,13 +20,13 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"github.com/stretchr/testify/mock"
-
 	v3 "github.com/tigera/api/pkg/apis/projectcalico/v3"
 	operatorv1 "github.com/tigera/operator/api/v1"
 	"github.com/tigera/operator/pkg/apis"
 	"github.com/tigera/operator/pkg/common"
 	"github.com/tigera/operator/pkg/controller/status"
 	"github.com/tigera/operator/pkg/controller/utils"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
same here - "appsv1"

Reply:
@ti-afra I think you missed renaming this one :)

Reply:
oops!